<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 130, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}

QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 110, 255);
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-1</y>
       <width>261</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/o.jfif);
background-color: rgb(85, 85, 255);
border-top-left-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>531</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>430</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border: 0px solid;
}
QPushButton:hover {
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>COMMENCER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 20pt &quot;Open Sans ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>Géoréférencement direct</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Système de cartographie mobile
– MMS -
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>271</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Réalisé par :
 TIHTIH Reda
BRIGUI Amina 

Encadré par :
Pr. Hakim BOULAASSAL 

Année universitaire : 2021 /2022</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/fst.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>61</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/res/uae.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Desktop/pppp/res.qrc"/>
 </resources>
 <connections/>
</ui>
